kaldb/src/test/java/com/slack/kaldb/writer/LogMessageWriterImplTest.java

```diff
@@ -90,46 +91,6 @@ private SearchResult<LogMessage> searchChunkManager(String indexName, String que
         Duration.ofMillis(3000));
   }
 
-  @Test
-  public void testApiLogMessageInsertion() throws Exception {
```

Comment:
removing this test - it's anyways of a depracted transformer i.e api transformer directly in the indexer